src/css/math.less

```diff
@@ -303,4 +303,28 @@
     cursor: text;
     font-family: @symbola;
   }
+
+  .mq-overarrow {
+    border-top: 1px solid black;
+    margin-top: 1px;
+    padding-top: 0.2em;
+
+    &:before {
+      display: block;
+      position: relative;
+      top: -0.34em;
+      font-size: 0.5em;
+      line-height: 0em;
+      content: '\27A4';
+      text-align: right;
+    }
+    &.mq-arrow-left:before {
+      -moz-transform: scaleX(-1);
+      -o-transform: scaleX(-1);
+      -webkit-transform: scaleX(-1);
+      transform: scaleX(-1);
+      filter: FlipH;
+      -ms-filter: "FlipH";
```

Comment:
I heard they considered including it, but were afraid there wasn't enough
space for the taco emoji

Han

On Thu, Mar 10, 2016 at 6:41 PM, Karol Tarasiuk <notifications@github.com>
wrote:

> In src/css/math.less
> <https://github.com/mathquill/mathquill/pull/544#discussion_r55783450>:
>
> > +    &:before {
> > +      display: block;
> > +      position: relative;
> > +      top: -0.34em;
> > +      font-size: 0.5em;
> > +      line-height: 0em;
> > +      content: '\27A4';
> > +      text-align: right;
> > +    }
> > +    &.mq-arrow-left:before {
> > +      -moz-transform: scaleX(-1);
> > +      -o-transform: scaleX(-1);
> > +      -webkit-transform: scaleX(-1);
> > +      transform: scaleX(-1);
> > +      filter: FlipH;
> > +      -ms-filter: "FlipH";
>
> Believe me or not, we couldn't find the opposite arrow which looked
> exactly the same...
>
> This is the right one:
> http://www.fileformat.info/info/unicode/char/27a4/index.htm It's called
> BLACK RIGHTWARDS ARROWHEAD. LEFTWARDS one doesn't exist...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mathquill/mathquill/pull/544/files#r55783450>.
>